[BUG] build faultinj failed spdlog/fmt/bundled/core.h: No such file or directory

**Describe the bug**
failed nightly build at commit https://github.com/NVIDIA/spark-rapids-jni/commit/32329a70c603dff117a69f500c1c2c6c035e5212
```
17:29:33  [INFO]      [exec] [ 58%] Building CUDA object CMakeFiles/spark_rapids_jni.dir/src/row_conversion.cu.o
17:29:33  [INFO]      [exec] [ 58%] Building CUDA object CMakeFiles/spark_rapids_jni.dir/src/zorder.cu.o
17:29:35  [INFO]      [exec] In file included from /home/jenkins/agent/workspace/jenkins-spark-rapids-jni_nightly-dev-352-cuda11/target/libcudf-install/include/spdlog/common.h:45,
17:29:35  [INFO]      [exec]                  from /home/jenkins/agent/workspace/jenkins-spark-rapids-jni_nightly-dev-352-cuda11/target/libcudf-install/include/spdlog/spdlog.h:12,
17:29:35  [INFO]      [exec]                  from /home/jenkins/agent/workspace/jenkins-spark-rapids-jni_nightly-dev-352-cuda11/src/main/cpp/faultinj/faultinj.cu:30:
17:29:35  [INFO]      [exec] /home/jenkins/agent/workspace/jenkins-spark-rapids-jni_nightly-dev-352-cuda11/target/libcudf-install/include/spdlog/fmt/fmt.h:27:14: fatal error: spdlog/fmt/bundled/core.h: No such file or directory
17:29:35  [INFO]      [exec]    27 | #    include <spdlog/fmt/bundled/core.h>
17:29:35  [INFO]      [exec]       |              ^~~~~~~~~~~~~~~~~~~~~~~~~~~
17:29:35  [INFO]      [exec] compilation terminated.
17:29:35  [INFO]      [exec] gmake[2]: *** [faultinj/CMakeFiles/cufaultinj.dir/build.make:76: faultinj/CMakeFiles/cufaultinj.dir/faultinj.cu.o] Error 1
17:29:35  [INFO]      [exec] gmake[1]: *** [CMakeFiles/Makefile2:1028: faultinj/CMakeFiles/cufaultinj.dir/all] Error 2
17:29:35  [INFO]      [exec] gmake[1]: *** Waiting for unfinished jobs....
17:32:11  [INFO]      [exec] [ 62%] Linking CXX shared library libcudf.so
17:32:50  [INFO]      [exec] [ 62%] Built target spark_rapids_jni
17:32:50  [INFO]      [exec] gmake: *** [Makefile:146: all] Error 2
```

--- a/src/main/cpp/faultinj/CMakeLists.txt
+++ b/src/main/cpp/faultinj/CMakeLists.txt
@@ -1,5 +1,5 @@
 #=============================================================================
-# Copyright (c) 2022, NVIDIA CORPORATION.
+# Copyright (c) 2022-2023, NVIDIA CORPORATION.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
@@ -22,20 +22,15 @@ project(
 
 set(CMAKE_CXX_STANDARD 17)
 
-find_package(Boost REQUIRED)
-
 add_library(
   cufaultinj SHARED
   faultinj.cu
 )
 
-find_path(SPDLOG_INCLUDE "spdlog"
-    HINTS "$ENV{RMM_ROOT}/_deps/spdlog-src/include")
-
-include_directories(
-  "${SPDLOG_INCLUDE}"
+target_link_libraries(
+  cufaultinj PRIVATE spdlog::spdlog_header_only
 )
 
 target_link_libraries(
-  cufaultinj CUDA::cupti_static
+  cufaultinj PRIVATE CUDA::cupti_static
 )

